porting/debug-build/wifi.rst

```diff
@@ -11,18 +11,24 @@ Testing wifi is done using NetworkManager
 
    nmcli d
 
-Qcom devices
-------------
+Qcom devices and CAF devices
+------------------------------------
 
-Wifi is fairly easy to get going on most Qcom based devices
+Wifi is fairly easy to get going on most Qcom and CAF based devices
```

Comment:
All CAF devices are Qcom-based, there is no need to mention CAF here imo

Also, I would prefer to see `WiFi` instead of `Wifi`, but it's not really related to your changes